tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -73,16 +73,22 @@ def self.available_options
                                        description: "Path to the BCSymbolMaps directory to build complete dSYM files",
                                        default_value: nil,
                                        optional: true,
-                                       verify_block: validate_symbol_maps)
+                                       verify_block: validate_symbol_maps),
+          FastlaneCore::ConfigItem.new(key: :project_root,
+                                       env_name: "BUGSNAG_PROJECT_ROOT",
+                                       description: "Root path of the project",
+                                       default_value: nil,
```

Comment:
By default, this value should be the current directory (`Dir.pwd`). This would avoid additional configuration in cases where the project was just built and the symbols are being uploaded.